.gitignore

```diff
@@ -1,14 +0,0 @@
-build
```

Comment:
This file is not associated with this pull request. Whether it is a misoperation?